src/dev/DevRelayClient.js

```diff
@@ -185,6 +195,16 @@ class DevRelayClient {
     return new BN(currentStake);
   }
 
+  async isRegistered(hub) {
+    let currentState;
+    try {
+      currentState = (await hub.methods.getRelay(this.relayerAddress).call()).state;
+    } catch (err) {
+      currentState = 0;
```

Comment:
I wouldn't fail silently here: we should at the very least log the error message.

Reply:
I see the same behavior [here](https://github.com/OpenZeppelin/openzeppelin-gsn-provider/blob/de551335f0c2a19aaa82c2b64968fdf100bf0e3f/src/dev/DevRelayClient.js#L177-L186). But I agree `console.log(err)` would be useful.